src/main/java/teammates/ui/controller/InstructorFeedbackEditSaveAction.java

```diff
@@ -52,11 +52,13 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                     + "<br><span class=\"bold\">Session visible from:</span> " + feedbackSession.sessionVisibleFromTime
                     + "<br><span class=\"bold\">Results visible from:</span> " + feedbackSession.resultsVisibleFromTime
                     + "<br><br><span class=\"bold\">Instructions:</span> " + feedbackSession.instructions;
+            data.setHasError(false);
         } catch (InvalidParametersException e) {
             setStatusForException(e);
+            data.setHasError(true);
         }
-        
-        return createRedirectResult(data.getInstructorFeedbackEditLink(courseId, feedbackSessionName));
+        data.setStatusForAjax(statusToUser.get(0));
```

Comment:
I believe that's because we store the status messages as an attribute in the HttpSession, which is not specific to just the current action.